index.ts

```diff
@@ -11,14 +11,20 @@ export interface CreateLoggerOptions {
   syslog?: boolean;
   metricsRegistry?: Registry;
   showDebug?: boolean;
+  systemdPrefix?: boolean;
```

Comment:
Do we have a readme to update?

Feels these should all be described.

Also not sure how well we bundle it, but JSDoc headers so we get tips for these props in vscode.

Reply:
It defaults to the env var, so you don't actually need to do anything, but yeah I'll see if there is anything to add to the readme

Reply:
Hmm, might not even expose it on the `create`, only really needed if you're doing things manually

Reply:
Oh I see the env var. Makes sense.

Should probably just put in the readme that it will automatically prefix severity information for systemd if detected by the presence of `process.env.JOURNAL_STREAM`